addons/website/static/src/snippets/s_countdown/options.js

```diff
@@ -22,11 +22,17 @@ options.registry.countdown = options.Class.extend({
      */
     endAction: function (previewMode, widgetValue, params) {
         this.$target[0].dataset.endAction = widgetValue;
-        if (widgetValue === 'message') {
+        if (widgetValue === 'message' || widgetValue === 'message_no_countdown') {
             if (!this.$target.find('.s_countdown_end_message').length) {
                 const message = this.endMessage || qweb.render('website.s_countdown.end_message');
                 this.$target.find('.container').append(message);
             }
+
+            if (widgetValue === 'message_no_countdown') {
+                this.$target.addClass('hide-countdown');
+            } else {
+                this.$target.removeClass('hide-countdown');
+            }
```

Comment:
One line with `toggleClass` + respect convention for the classname (o_) + find a better name